pyproject.toml

```diff
@@ -1,3 +1,9 @@
+[project]
+name = "doctor"
+version = "1"
+description = "CourtListener doctor"
+requires-python = ">=3.10"
```

Comment:
I realized for this repository that I needed to declare the minimum Python version for the pyupgrade rules to know which syntax to upgrade. I'll now add this metadata to repositories as I add Ruff, and for the previously Ruff-migrated repos, I'll add it when I set up uv.